CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 v0.8.0
 =====
+- Add configuration for iOS layout
```

Comment:
Sorry to be nitpicky... can we mention something that the configuration is selecting drawer layout or tabs?